src/client/sandbox/node/element.ts

```diff
@@ -502,9 +502,15 @@ export default class ElementSandbox extends SandboxBase {
 
             formSubmit () {
                 sandbox._ensureTargetContainsExistingBrowsingContext(this);
-                sandbox.emit(sandbox.BEFORE_FORM_SUBMIT_EVENT, { form: this });
 
-                return nativeMethods.formSubmit.apply(this, arguments);
+                const args = { form: this, preventSubmit: false };
```

Comment:
Could you please add a `// HACK: ` comment with a short description and a link to the corresponding TestCafe issue?